### DISCUSSION TITLE
how to ignore trace in  server-side

### DISCUSSION THREAD

**Discussion by darknesstm** [apache/skywalking#10585] [Q&A] [answered]
Is it currently possible to actively ignore very short-duration trace or specific path trace in server-side configuration?

**Comment by wu-sheng** [comment, marked as answer]
If the trace breaking is acceptable, red this doc https://skywalking.apache.org/docs/main/next/en/setup/backend/trace-sampling/

**Reply by darknesstm** [reply]
However, the documentation only provides the setting for sampling rate and does not provide how to perform specific exclusions. I do not need random filtering. In fact, I hope to uniformly filter out some heartbeat requests on the server side instead of configuring them separately on the client side.

**Reply by wu-sheng** [reply]
That should be done at agent side.

**Reply by darknesstm** [reply]
usually business developers within an organization are not very familiar with the configuration of middleware. If it is possible to override client-side configuration with server-side configuration, it would be more suitable from a management perspective.

**Reply by wu-sheng** [reply]
No, it doesn't. You don't count the cost of resources. If we do that on the server side, it costs huge CPUs. It is unreasonable.

> usually business developers within an organization are not very familiar with the configuration of middleware

How to deploy agent is relying on INFRA team, there is nothing about dev team.

**Reply by darknesstm** [reply]
What I mean is to influence the agent-side configuration by issuing high-priority configuration from the server-side, which will not increase the CPU usage of the server. Of course, I understand what you mean, but in **some** organizations, the infrastructure team may only provide agent client configuration samples and the server ,  cannot exercise complete control over them.

**Reply by wu-sheng** [reply]
I am not going to persuade you. That is your choice. I am just representing what the community thinks, and why we don't have that.
Please understand that. We don't need to be alignment.
